README.md

```diff
@@ -1 +1,78 @@
-# codacy-test
\ No newline at end of file
+# Codacy ESLint
+
+## Adding new plugins / configs
+
+1.  Install the package using npm
+2.  \[Plugins only\] Add the plugin to the plugins section in the file `src/eslintDefaultOptions.ts`
```

Comment:
Codacy has a fix for the issue: [Expected: 1; Actual: 2](https://app.codacy.com/gh/franciscodua/codacy-test/pullRequest?prid=7573067)
```suggestion
3. \[Plugins only\] If the plugin has descriptions for rules on GitHub, reference them
```